platform/darwin/src/MGLRasterSource.h

```diff
@@ -14,6 +14,22 @@ NS_ASSUME_NONNULL_BEGIN
  raster source is added to an `MGLStyle` object along with an
  `MGLRasterStyleLayer` object. Use a raster style layer to control the
  appearance of content supplied by the raster source.
+ 
+ ### Example ###
+ 
+ ```swift
+ // Define tileset
+ let tileset = MGLTileSet(tileURLTemplates: ["https://example.com/raster-tiles/{z}/{x}/{y}.png"])
```

Comment:
#7377 removed MGLTileSet in favor of direct source initializers with option dictionaries. Sorry to keep shifting the goalposts on you. There’s still an attribution option that takes an HTML string, but now there’s also an option that takes structured MGLAttributionInfo objects.